### ISSUE TITLE
Question about loss computing

### ISSUE THREAD

**Issue by RogerXT** [OpenNMT/OpenNMT-py#977]
In onmt.utils.loss.NMTLossCompute._compute_loss, 

loss = self.criterion(scores, gtruth)

I know that we can get pred from scores, and gtruth is the target.

I am trying to know what gtruth contains. A batch of sequences? Or, a single sequence, and the sequence contains a batch of sentences?
